include/l0_sampling/sketch.h

```diff
@@ -50,30 +57,30 @@ class Sketch {
    * Construct a sketch of a vector of size n
    * @param n Length of the vector to sketch. (static variable)
    * @param seed Seed to use for hashing operations
-   * @param num_bucket_factor Factor to scale the number of buckets in this sketch (static variable)
+   * @param failure_factor The rate at which an individual sketch is allowed to fail (determines column width)
```

Comment:
besides seed, these aren't params for the constructor. put them where they are used (maybe config?)

Reply:
Either that or make it clear that these are the list of possible params for all constructors/factory methods

Reply:
Should be done now